packages/core/src/ChatHistory.ts

```diff
@@ -178,7 +187,10 @@ export class SummaryChatHistory extends ChatHistory {
     const requestMessages = this.calcCurrentRequestMessages(transientMessages);
 
     // get tokens of current request messages and the transient messages
-    const tokens = this.llm.tokens(requestMessages);
+    const tokens = requestMessages.reduce(
```

Comment:
@himself65 this is just calculating the tokens of the `message.content` but not the used tokens for a message prompt like in the old code